type.go

```diff
@@ -8,8 +8,29 @@ import (
 	"strconv"
 	"sync"
 	"time"
+	"strings"
 )
 
+type Parameter struct {
+	distance int
+	latitude float64
+	longitude float64
```

Comment:
Perfect fit for a `Coord` here instead :D 
```suggestion
	startingPoint Coord
```